### PR TITLE
BUG: Patch Headers in OStreamUtilities

### PR DESCRIPTION
**Pull request by nyoungbq**
Broken merge overwrote crucial change

### CODE REVIEW

File: src/complex/Utilities/OStreamUtilities.cpp [hunk 1]
```diff
@@ -299,7 +299,14 @@ class TupleWriter : public ITupleWriter
...
     {
-      outputStrm << m_DataArray.getName() << "_" << index;
+      if(index == 0)
+      {
+        outputStrm << m_DataArray.getName();
+      }
+      else
+      {
+        outputStrm << m_DataArray.getName() << "_" << index;
+      }
```
Comment (imikejackson) [inline src/complex/Utilities/OStreamUtilities.cpp]: The issue here is that if there are 2 components then the first will be "Foo" while the second will be "Foo_1". To be clear what should happen is if there is only 1 component then the output should be:
`Foo`
if there are multiple components then the output should be:
`Foo_1,Foo_2`
 
Apologies for not being clear enough.

**Review by imikejackson — APPROVED** [review]

File: src/complex/Utilities/OStreamUtilities.cpp (hunk 1 above)
Comment (nyoungbq) [reply src/complex/Utilities/OStreamUtilities.cpp]: Oh ok lll get that patched

Comment (imikejackson) [reply src/complex/Utilities/OStreamUtilities.cpp]: Waiting on the patch to re-approve.

**Review by imikejackson — CHANGES_REQUESTED** [review]

File: src/complex/Utilities/OStreamUtilities.cpp (hunk 1 above)
Comment (imikejackson) [reply src/complex/Utilities/OStreamUtilities.cpp]: I so screwed that up... It _should_ have been:

if there are multiple components then the output should be:
```
Foo_0,Foo_1,Foo_2....
```

Comment (imikejackson) [reply src/complex/Utilities/OStreamUtilities.cpp]: I just pushed the fix but that will screw up the complex_plugins PR.

**Pull request closed without merging**